modules/administration-guide/examples/che-prepulled-images-table.adoc

```diff
@@ -22,4 +22,26 @@
 |che-nodejs10-ubi |quay.io/eclipse/che-nodejs10-ubi:{prod-ver} |{prod-ver}
 
 |che-quarkus |quay.io/eclipse/che-quarkus:{prod-ver} |{prod-ver}
-|===
\ No newline at end of file
+|===
+
+[NOTE]
+====
+The table of recommended images to pre-pull is updated once per release and might vary from the most updated build at the time.
+
+To get the latest build:
+
+. Use the following scriptlet:
++
+[subs="+attributes,+quotes"]
+----
+$➔ curl -sSLo- https://raw.githubusercontent.com/eclipse/che-operator/master/olm/eclipse-che-preview-openshift/deploy/olm-catalog/eclipse-che-preview-openshift/{che-ver}/eclipse-che-preview-openshift.v{che-ver}.clusterserviceversion.yaml | yq -r '.spec.relatedImages[].image' | sort | uniq
+----
++
+. Exclude redundant, unused digests, and unsupported images for OpenShift pre-pulling such as:
++
+* machine-exec image
+* images without a sleep command installed
+* images that mount volumes in the docker file
++
+For additional information, see link:https://github.com/eclipse/che/issues/17562[GitHub Issue 17562]
```

Comment:
Note that old versions of the same image will be removed soon. See https://github.com/eclipse/che-plugin-registry/pull/672

If you wait to merge this PR until after Che 7.22.0 is live, the instructions will be simpler.

Reply:
OK, will prepare your suggestion and wait for the 7.22.0